spacehammer/config.example.fnl

```diff
@@ -1,12 +1,12 @@
-(require-macros :lib.macros)
-(require-macros :lib.advice.macros)
-(local windows (require :windows))
-(local emacs (require :emacs))
-(local slack (require :slack))
-(local vim (require :vim))
+(require-macros :spacehammer.lib.macros)
+(require-macros :spacehammer.lib.advice.macros)
```

Comment:
Are these used at all? They'd have to be used through the fully name, and anyway we should use `import-macros`.